### DISCUSSION TITLE
Resource state is never set to `Stopping`.

### DISCUSSION THREAD

**Discussion by slashkeyvalue** [citizenfx/fivem#1598] [Engineering Discussion]
Self explanatory. noticed while using `GET_RESOURCE_STATE`, it has a case for the `Stopping` state but its never actually set by the resourceimpl.

**Comment by blattersturm** [comment]
I believe this might've been a case of 'the enumeration was defined before the implementation was'. Commits related to this subsystem are mostly from 2015, so I'm not sure if anyone is aware of the situation there.

GET_RESOURCE_STATE meanwhile just handles all cases that exist in the enumeration.
